daily-challenges/2024-09/25/ththao27.spec.js

```diff
@@ -0,0 +1,47 @@
+import { test, chromium, expect } from '@playwright/test';
+
+// Javascript
+
+function trimSpaces(str) {
+    return str.split(/\s+/).join(' ');
+}
+
+// output 1
+
+const str1= "   Xin   chào   mọi người   ";
+console.log(trimSpaces(str1));
+
+// output 2
+
+const str2= "   JavaScript    is    fun   ";
+console.log(trimSpaces(str2));
+
+// output 3
+
+const str3= "   Lập  trình   JavaScript   ";
+console.log(trimSpaces(str3));
+
+// Playwright
+
+test('ththao27 - 2024-09 day 25: Emulate permissions for location, camera, and microphone', async ({ }) => {
+  const browser = await chromium.launch({headless: false});
```

Comment:
Not a big problem, but có thể dùng page như bình thường, không cần tạo mới context.
Tham khảo cách ở file của playwrightvietnam.spec.ts